app/models/completion_result.rb

```diff
@@ -0,0 +1,6 @@
+# Represents a set of query completion results for an autocomplete feature
+class CompletionResult
+  include ActiveModel::Model
+
+  attr_accessor :suggestions
+end
```

Comment:
This model looks so basic that you might just want it to be a nested class in the Complete namespace. e.g: `CompletionResult = Data.define(:suggestions)`. 

No worries though if it's got a more complex future looming.

Reply:
It's super basic, and I love `Data.define` much more than I do ActiveModel, but we've stuck with these for the data classes because `ActiveModel` gives us free serialisation and it's a bit more Rails-y:

```ruby
completion_result = CompletionResult.new(suggestions: [...])
render json: completion_result
```

Reply:
Ah yeah that makes sense 👍